specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2018-11-19/machineLearningServices.json

```diff
@@ -1153,6 +1232,24 @@
       },
       "description": "The List Usages operation response."
     },
+    "ListUsagesByVMFamilyResult": {
+      "properties": {
+        "value": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/UsageByVMFamily"
+          },
+          "description": "The list of compute resource usages, by VM family, broken down by Workspace, and Cluster usage."
+        },
+        "nextLink": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The URI to fetch the next page of compute resource usage information. Call ListNext() with this to fetch the next page of compute resource usage information."
```

Comment:
Not all SDKs will have a method named `ListNext()`, please reword so it's generic.

Reply:
Will just this be enough:
"The URI to fetch the next page of compute resource usage information"

Reply:
Yes sounds good to me.